lib/fog/google/models/storage_json/directories.rb

```diff
@@ -0,0 +1,37 @@
+require "fog/core/collection"
+require "fog/google/models/storage_json/directory"
+
+module Fog
+  module Storage
+    class GoogleJSON
+      class Directories < Fog::Collection
+        model Fog::Storage::GoogleJSON::Directory
+
+        def all
+          data = service.get_service.body["items"]
+          load(data)
+        end
+
+        def get(key, options = {})
+          remap_attributes(options,             :delimiter  => "delimiter",
+                                                :marker     => "marker",
+                                                :max_keys   => "max-keys",
+                                                :prefix     => "prefix")
+          data = service.get_bucket(key, options).body
+          directory = new(:key => data["name"])
+          # options = {}
```

Comment:
I'm not really sure. This is one of the places where the JSON API diverges from the XML one and I'm a little confused about what the expected behavior is.

First: none of those keys or anything similar exist in the [Bucket resource](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource) that get_bucket returns. So that loop can be ditched.

Second, however, I'm not sure what the significance of the `directory.files.load(data["contents"])` is. `contents` isn't a valid key in the Bucket resource either, so I doubt this would help us in any way.

It _seems_ like we can ditch this whole block from that. I left it in because I wasn't entirely sure.